### PR TITLE
DOP-4281: Button to Open Instruqt Labs

### PR DESCRIPTION
**Pull request by anabellabuckvar**
### Stories/Links:

[DOP-4281](https://jira.mongodb.org/browse/DOP-4281)

### Current Behavior:

[Prod](https://www.mongodb.com/docs/manual/tutorial/getting-started/)

### Staging Links:

[staging with button](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/drawers/DOP-4280/docs/anabella.buckvar/DOP-4281/tutorial/getting-started/index.html
)
[staging without button](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/DOP-4280/docs/anabella.buckvar/DOP-4281/tutorial/getting-started/index.html)

### CODE REVIEW

**Comment by caesarbell** [comment]
@anabellabuckvar , correct me if I am wrong, but the interactive lab should only appear if the button was clicked, if that is correct, I am not seeing that behavior in your [staging with button](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/DOP-4280/docs/anabella.buckvar/DOP-4281/tutorial/getting-started/index.html) example. 

Also, your [staging without button](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/DOP-4280/docs/anabella.buckvar/DOP-4281/tutorial/getting-started/index.html) example is also showing a button.

**Review by caesarbell — APPROVED** [review]
LGTM. I just left a comment.

**Comment by anabellabuckvar** [comment]
> @anabellabuckvar , correct me if I am wrong, but the interactive lab should only appear if the button was clicked, if that is correct, I am not seeing that behavior in your [staging with button](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/DOP-4280/docs/anabella.buckvar/DOP-4281/tutorial/getting-started/index.html) example.
> 
> Also, your [staging without button](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/DOP-4280/docs/anabella.buckvar/DOP-4281/tutorial/getting-started/index.html) example is also showing a button.

Thanks for the LGTM and comments! The staging links should be fixed now, I didn't know how to create two versions of the same staging link beforehand.

As far as the interactive lab only appearing if the button is clicked, the interactive lab should either be present in page content if the drawer option is false, or open up after clicking the button as a drawer if the drawer option is true. There is a different PR in which Raymund created the drawer and dealt with the conditionality, so I chose to just leave the code for the Lab content itself as is in my PR.

**Review by rayangler — COMMENTED** [review]
Just wanted to leave a comment about some extra css. It's non-blocking though. Otherwise looks good

File: src/components/Heading.js [hunk 1]
```diff
@@ -62,6 +71,17 @@ const Heading = ({ sectionDepth, nodeData, page, ...rest }) => {
             return <ComponentFactory {...rest} nodeData={element} key={index} />;
           })}
           <Permalink id={id} description="heading" />
+          {shouldShowLabButton && (
+            <Button
+              role="button"
+              className={cx(labButtonStyling)}
+              disabled={isOpen}
+              onClick={() => setIsOpen('True')}
```
Comment (rayangler) [inline src/components/Heading.js:79]: We should keep this a boolean `true`

**Review by rayangler — COMMENTED** [review]
Pretty much looks good. Just one last change

**Pull request merged**